packages/truffle-core/lib/command.js

```diff
@@ -14,9 +14,13 @@ function Command(commands) {
   });
 
   this.args = args;
-};
+}
+
+Command.prototype.getCommand = function(inputStrings, config, noAliases) {
```

Comment:
We can just get rid of `noAliases` because it comes from `config` anyway.

I think we can even get away with doing `function (inputStrings, options)` and doing `Config.detect()` inside the function here. There might be a small performance loss, but we can always solve that inside `detect` itself.